index.html

```diff
@@ -3874,7 +3874,58 @@ <h3>Syntactic Sugar</h3>
             </li>
           </ul>
         </section>
-
+        <section>
+          <h3>Lists and Arrays</h3>
+          <p>
+Lists and arrays, even lists of lists, are possible when using [[JSON-LD]] 1.1
+as this specification does. We encourage those who want RDF semantics with use
+cases requiring lists and arrays to follow the guidance on
+<a href="https://www.w3.org/TR/json-ld11/#lists">lists in JSON-LD 1.1</a>.
+          </p>
+          <p>
+In general, a JSON array is ordered, while a JSON-LD array is not ordered unless
+that array uses the <code>@list</code> keyword.
+          </p>
+          <p class="note">
+While it is possible to use this data model without any JSON-LD processing,
+those who do so and make use of arrays need to be aware that unless the above
+guidance is followed, the order of items in an array cannot be guaranteed in
+JSON-LD. This may lead to unexpected behavior.
+          </p>
+          <p>
+If JSON structure or ordering is important to your application, 
+we recommend you mark such elements as `@json` via an `@context`.
+          </p>
+          <pre class="example" title="A verifiable credential with an embedded JSON data structure">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2",
+    { 
```

Comment:
We should provide the markup snippet separately from the example. We don't want people defining this sort of stuff in-line as it'll probably harm interop (because people will cut/paste JSON-LD Context stuff instead of just cut/pasting URLs that can be cached w/ known hashes).

Reply:
Yes, please.

Reply:
attemped in [b5ec488](https://github.com/w3c/vc-data-model/pull/1258/commits/b5ec488a33b2b2c3b2cc0774bd1fa5ee6b412313)